docs/examples/DS-82--form.js

```diff
@@ -0,0 +1,262 @@
+                expandUnderClassNames: 'schemaform-expandUnder-indent'
+              }
+            },
+            'view:bookType': {
+              'ui:options': {
+                expandUnder: 'view:book'
+              },
+              'ui:description': 'Please choose the passport book type. The large book option is for those who frequently travel abroad during the passport validity period, and is recommended for applicants who have previously required the addition of visa pages.'
+            },
+            'card': {
+              'ui:options': {
+                expandUnderClassNames: 'schemaform-expandUnder-indent'
+              }
+            },
+            'view:cardWarning': {
+              'ui:options': {
+                expandUnder: 'card'
+              },
+              'ui:description': 'Note that a U.S. passport card is only valid for land or sea travel to and from Canada, Mexico, and the Carribean. For international air travel you must use a passport book.'
+            }
+          },
+          schema: {
+            type: 'object',
+            required: [ ],
+            properties: {
+              'view:bookInstructions': {
+                type: 'object',
+                properties: {}
+              },
+              'view:book': {
+                type: 'boolean',
+                title: 'U.S. passport book'
+              },
+              'view:bookType': {
+                type: 'object',
+                properties: {
+                  book: {
+                    type: 'string',
+                    title: 'Passport book type',
+                    'enum': [ 'Regular (standard) book', 'Large (non-standard) book']
+                  },
+                }
+              },
+              card: {
+                type: 'boolean',
+                title: 'U.S. passport card'
+              },
+              'view:cardWarning': {
+                type: 'object',
+                properties: {}
+              }
+            }
+          }
+        }
+      }
+    },
+    CP3: {
+      title: "Applicant information",
+      pages: {
+        PG3: {
+          path: 'applicant-info',
+          title: 'Applicant information',
+          uiSchema: {
+            usedOtherNames: {
+              "ui:options": {
+                expandUnderClassNames: "schemaform-expandUnder-indent"
+              }
+            },
+            otherNames: {
+              'ui:widget': 'textarea',
+              "ui:options": {
+                 expandUnder: "usedOtherNames"
+               }
+            },
+            phone: {
+              'ui:errorMessages': {
+                minLength: 'Please enter a phone number, including area code'
+              }
+            },
+            ssn: {
+              'ui:errorMessages': {
+                pattern: 'Please enter a social security number in the form 000-00-0000'
+              }
+            },
+            dob: currentOrPastDate('Date of birth')
```

Comment:
I assume this is the documentation for version 1.1.0, because the definition files have been changed since then, but not yet released.

Reply:
Yes, I agree it would be nice not to have to go back. Perhaps we should decide definitely when we're releasing the new definitions; if it's going to be next week we might as well update the docs to reflect that now.

Reply:
Since we didn't pull in the definition changes for 1.2.0 I am going back to the old style.